arcade/text.py

```diff
@@ -613,6 +613,42 @@ def position(self, point: Point):
         else:
             self._label.position = x, y, self._label.z
 
+    @property
+    def tracking(self) -> float | None:
+        """
+        Get the tracking amount for this text object, or rather,
+        the added space between each character.
+
+        Returns a pixel amount, or None if the tracking is inconsistent.
+        """
+        kerning = self._label.get_style("kerning")
+        return kerning if kerning != pyglet.text.document.STYLE_INDETERMINATE else None
+
+    @tracking.setter
+    def tracking(self, value: float):
+        """
+        Get the tracking amount for this text object, or rather,
+        the added space between each character.
+
+        `value` is an amount in pixels and can be negative.
+        To convert from the em unit, use Text.em_to_px().
+        """
```

Comment:
The setter docstring is never visible in docs. It needs to be merged with property. "Get or set" is a decent indication it's not a read-only property.